data-serving/data-service/src/index.ts

```diff
@@ -32,17 +32,14 @@ app.use('/api', apiRouter);
     try {
         console.log(
             `  Connecting to instance ${env.DB_CONNECTION_STRING}
-               with db ${env.DB_NAME}
-               and collection ${env.DB_COLLECTION}`,
+               with db ${env.DB_NAME}`,
         );
 
-        const client = new DbClient(env.DB_CONNECTION_STRING);
-        await client.connect();
+        await mongoose.connect(`${env.DB_CONNECTION_STRING}/${env.DB_NAME}`, {
```

Comment:
do you want to follow the same approach as I did and merge those two envs vars into one? I don't see the benefit of splitting them.

Reply:
SG, updated.